### ISSUE TITLE
Unexpected token while processing with ecmascript

### ISSUE THREAD

**Issue by kimchirichie** [sebakerckhof/ufs-s3#4]
after adding the package:

```
$ meteor
=> Errors prevented startup:                  

   While processing files with ecmascript (for target web.browser):
   packages/seba:ufs-s3/ufs-s3.js:161:16: Unexpected token (161:16)

   While processing files with ecmascript (for target os.linux.x86_64):
   packages/seba:ufs-s3/ufs-s3.js:161:16: Unexpected token (161:16)

=> Your application has errors. Waiting for file change.
```

apparently 161 has a comma missing.
source code on git actually shows the comma is there. :S

**Comment by sebakerckhof** [comment]
Yeah, this is quite quickly ported from collectionfs. But apparently they monkey patch the S3 library.
However, I'm currently not set-up to test it. Could you maybe test the latest commit locally (cloning the package into the 'packages' folder) and see if that solves the issue?

**Comment by dufia** [comment]
I guess, its still broken?

**Comment by mackbrowne** [comment]
I'm having the same problem.  @dufia was there a solution that isn't listed here?  Is that why it was closed?

**Comment by dufia** [comment]
@mackbrowne 

There is no solution, I've used jalik:ufs-local. If you choose to use it, get ready for another set of challenges.

**Comment by mackbrowne** [comment]
I've given up on the jalik stack.  I've used the gridFS one before, but i've finally decided to stick with meteor-slingshot and do my transformations like resizing and cropping from the client side.

Please mark this repository as Dead, Do not Use or something similar.  Its a shame really, but if nobody is using / contributing to this, its the responsible thing to do.
